nomad/alloc_endpoint_test.go

```diff
@@ -1284,8 +1284,9 @@ func TestAllocEndpoint_List_AllNamespaces_ACL_OSS(t *testing.T) {
 		{
 			Label:     "all namespaces with insufficient token",
 			Namespace: "*",
-			Allocs:    []*structs.Allocation{},
 			Token:     ns1tokenInsufficient.SecretID,
+			Error:     true,
+			Message:   structs.ErrPermissionDenied.Error(),
```

Comment:
This was the only test failure after this refactoring, but I think this is the right answer? The token used doesn't have the required `read-job` permission, so it should return a `403` rather than an empty list.